docs/context/ref/Message.md

```diff
@@ -0,0 +1,78 @@
+---
+id: Message
+sidebar_label: Message
+title: Message
+hide_title: true
+---
+# `Message`
+
+A chat message to be sent through an instant messaging application. Can contain one or several text bodies (organised by mime-type, plaintext or markdown), 
+as well as attached entities (either arbitrary file attachments or fdc actions to be embedded in the message). To be put inside a ChatInitSettings object.
+
+## Type
+
+`fdc3.message`
+
+## Schema
+
+https://fdc3.finos.org/schemas/next/message.schema.json
+
+## Details
+
+| Property          | Type                                      | Required | Example Value           |
+|-------------------|-------------------------------------------|----------|-------------------------|
+| `type`            | string                                    | Yes      | `'fdc3.message'`   |
+| `text`            | map of string content to string mime-type | No       | { text/plain: 'Hello' } |
```

Comment:
```suggestion
| `text`            | map of string mine-type to string content | No       | { text/plain: 'Hello' } |
```

Also we need a reference to an external standard for mime-types (should also get added to references page)

Reply:
https://www.iana.org/assignments/media-types/media-types.xhtml